2020/12/23 09:12:20 : Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 665, in storage_data
    getattr(self.get_source_cart(), 'finish_storage_data')()
  File "/var/www/python/cartmigration/models/basecart.py", line 3929, in finish_storage_data
    folder_upload = self.get_folder_upload(self._migration_id)
  File "/var/www/python/cartmigration/models/basecart.py", line 3941, in get_folder_upload
    folder_upload = root_path + '/' + DIR_UPLOAD + '/' + get_config_ini('src', 'file', file = 'test.ini')
TypeError: can only concatenate str (not "NoneType") to str

2020/12/23 09:12:23 : categories_errors 2: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:23 : categories_errors 3: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:23 : categories_errors 4: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:23 : categories_errors 5: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:23 : categories_errors 6: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:23 : categories_errors 7: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:23 : categories_errors 8: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:23 : categories_errors 9: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:24 : categories_errors 10: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:24 : categories_errors 11: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:24 : categories_errors 12: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:24 : categories_errors 13: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:24 : categories_errors 14: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:24 : categories_errors 15: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:24 : categories_errors 16: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:24 : categories_errors 17: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:25 : categories_errors 19: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:25 : categories_errors 20: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:25 : categories_errors 22: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

2020/12/23 09:12:25 : categories_errors 23: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/var/www/python/cartmigration/models/cart/woocommerce.py", line 2328, in category_import
    return response_success(category_term_import)
NameError: name 'category_term_import' is not defined

